pkg/chunk/storage/utils_test.go

```diff
@@ -40,3 +40,27 @@ func forAllFixtures(t *testing.T, storageClientTest storageClientTest) {
 		})
 	}
 }
+
+func forAllTableFixtures(t *testing.T, storageClientTest storageClientTest) {
+	var fixtures []testutils.Fixture
+	fixtures = append(fixtures, aws.Fixtures...)
+	fixtures = append(fixtures, gcp.Fixtures...)
+	fixtures = append(fixtures, local.Fixtures...)
+	fixtures = append(fixtures, Fixtures...)
+
+	cassandraFixtures, err := cassandra.Fixtures()
+	require.NoError(t, err)
+	fixtures = append(fixtures, cassandraFixtures...)
+
+	for _, fixture := range fixtures {
+		if fixture.Type() == testutils.Table {
```

Comment:
I can't figure out why this check is required. Every storage type has an index, which is some kind of table.

Reply:
Object store backends will successfully store a chunk even if the schema doesn't have a chunk table present the timestamp. As a result this test was really only relevant for chunk stores that use a schema. This is only needed to test the behavior of storing a chunk when there is no relevant schema. Since this adds testing complexity without much benefit I'll remove it.